src/pretix/control/templates/pretixcontrol/event/mail.html

```diff
@@ -23,7 +23,29 @@ <h4 class="panel-title">
                     </div>
                     <div id="order_placed" class="panel-collapse collapse">
                         <div class="panel-body">
-                            {% bootstrap_field form.mail_text_order_placed layout="horizontal" %}
+                            <div id="mail_text_order_placed_panel" class="preview-panel" for="mail_text_order_placed">
+                                <label class="col-md-3 control-label">{% trans "Text" %}</label>
+                                <div class="col-md-9">
+                                    <div class="tab-content">
+                                        <div id="mail_text_order_placed_edit" class="tab-pane fade in active">
+                                            {% bootstrap_field form.mail_text_order_placed show_label=False %}
+                                        </div>
+                                        <div id="mail_text_order_placed_preview" class="tab-pane">
+                                            {% for l in request.event.settings.locales %}
+                                                <pre lang="{{ l }}" for="mail_text_order_placed_{{ forloop.counter0 }}"></pre>
+                                            {% endfor %}
+                                        </div>
+                                    </div>
+                                    <ul class="nav nav-pills pull-right">
+                                        <li role="presentation" class="active">
+                                            <a data-toggle="pill" type="edit" href="#mail_text_order_placed_edit"><i class="fa fa-pencil-square-o fa-fw"></i> {% trans "Edit" %}</a>
+                                        </li>
+                                        <li role="presentation">
+                                            <a data-toggle="pill" type="preview" href="#mail_text_order_placed_preview"><i class="fa fa-tv fa-fw"></i> {% trans "Preview" %}</a>
+                                        </li>
+                                    </ul>
+                                </div>
+                            </div>
```

Comment:
Just a thought: We will need all of this boilerplate for every single mail template. I think once the basic setup is clear, we should put this into a seperate template (i.e. ``mail_settings_fragment.html``) that we [{% include %}](https://docs.djangoproject.com/en/1.10/ref/templates/builtins/#include).